app/data/mounts.json

```diff
@@ -632,6 +734,20 @@
             "spellid": 336064,
             "icon": "inv_horse2bastionmount_purple",
             "itemId": 181317
+          },
+          {
+            "ID": 1367,
+            "name": "Chosen Tauralus",
```

Comment:
Achievement rewards should go in the achievement section for consistency reasons.

Reply:
I think we need some sort of rule on where to put stuff with shadowlands mounts. This mount does come from an achievement where you need a specific covenant (necrolords) and is part of the covenant sanctum. If we put this under achievements we also need to move other mounts. For example, some mounts from rare mobs drop only for specific covenants.

I think we can do it two ways:

1. Keep the old order, there are no covenant sections. Every mount goes into quest, achievement, drop, whatever.
2. Move every mount that has a covenant requirement under its specific covenant where it does not matter if it is a drop, achievement, ...

Based on that I would redo this PR since a lot more stuff is uncovered now and the old data is not even valid anymore.

Reply:
Yeah, I think we should go with 2. I hadn't realized this mount was covenant-specific.

Reply:
Covenant specific mounts are now under their covenants. Mounts that drop from rare spawns that only require covenant interaction are still under rare spawn, because they still can be looted by everyone.